Makefile

```diff
@@ -1,6 +1,11 @@
-.PHONY: image test docs bundle
+.PHONY: image test docs bundle release
 
 IMAGE_NAME ?= codeclimate/codeclimate-rubocop
+RELEASE_REGISTRY ?= codeclimate
+
+ifndef RELEASE_TAG
+override RELEASE_TAG = latest
+endif
```

Comment:
```suggestion
RELEASE_TAG ?= latest
```

Reply:
@fede-moya this sadly does not work when providing an empty string. Makefile takes empty strings as not null, so it'll end up not tagging the image as `latest`. The `ifndef` workaround lets us provide an empty string that will still be replaced with `latest`